modules/nw-networkpolicy-about.adoc

```diff
@@ -11,7 +11,10 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
 
 [NOTE]
 ====
-IPBlock is supported by network policy with limitations for OpenShift SDN; it supports IPBlock without except clauses. If you create a policy with an IPBlock section that includes an except clause, the SDN pods log warnings and the entire IPBlock section of that policy is ignored.
+When using the OpenShift SDN cluster network provider, the following limitations apply regarding network policies:
+
+* Egress network policy as specified by the `egress` field is not supported.
+* IPBlock is supported by network policy with limitations for OpenShift SDN; it supports IPBlock without `except` clauses. If you create a policy with an IPBlock section that includes an `except` clause, the SDN pods log warnings and the entire IPBlock section of that policy is ignored.
```

Comment:
_IPBlock is supported by network policy with limitations for OpenShift SDN; it supports IPBlock without `except` clauses._
Can these two sentences be merged into one? It seems a bit clunky here. 
_Network policy for OpenShift SDN only supports IPBlock without `except` clauses._ (Or something like that)
#wordsarehard